src/control/server/config/server.go

```diff
@@ -418,6 +412,25 @@ func (c *Server) Validate(log logging.Logger) (err error) {
 		}
 	}()
 
+	// For backwards compatibility, allow specifying "servers" rather than
+	// "engines" in the server config file.
+	if len(c.Servers) > 0 {
```

Comment:
I think there should also be a deprecation warning here, to the effect that the config needs to be updated before the next release. I don't think we want to carry support for both styles indefinitely.

Reply:
agreed.

Reply:
https://jira.hpdd.intel.com/browse/DAOS-6758

Reply:
fixed in #4611